src/nidcpower/metadata/enums_addon.py

```diff
@@ -179,6 +179,10 @@
         ],
     },
     'PowerLineFrequency': {}, # Enum metadata actually contains constants.
```

Comment:
It'd be better to keep the comments consistent. For example, in src/nidcpower/metadata/attributes_addon.py you have "# POWER_LINE_FREQUENCY" but a comment like this one could apply perfectly fine.

Also:
"# Disconnecting enum since it is a boolean" vs " # Delete because boolean values only"

Reply:
Made comments consistent across drivers